### ISSUE TITLE
Visual Studio Port

### ISSUE THREAD

**Issue by Hades32** [wende/autocomplete-elixir#40]
After it becoming open-source, I have played around with Visual Studio Code, and man it is far better than Atom. Especially more responsive, but also better usability IMHO.

So now your plugin is the only reason for me still using Atom :)

I had a look at the VS Code plugin for Go [1] (which seems to have a comparable way of implementing code completion with invoking external application like you) and it looked far easier than Atom's...

Maybe have a look if you find it worth to port from Atom to VS Code

[1] https://github.com/Microsoft/vscode-go

**Comment by wende** [comment]
First of all that has just made my day :grinning: 
I'm glad it's useful to someone 

And second If VSC allows running another process in the background then I guess it shouldn't be hard because the very heart of the plugin is an Elixir script file. I'll take a look 

Merry Christmas :gift:

**Comment by Hades32** [comment]
Well, looking at https://atom.io/packages/autocomplete-elixir it's useful to a few thousand people! :grinning: 

But being serious, this plugin finally made me productive with Elixir! I just cannot live without any form of autocomplete, and I'm not willing to stay with vim or emacs!

So, in any case, thank you very much and also a Merry Christmas to you! :christmas_tree:

**Comment by devinshively** [comment]
:+1:

**Comment by Hades32** [comment]
VSCode recently reached 1.0 and thus should provide a stable API.

**Comment by mat-mcloughlin** [comment]
@wende yep its does allow out of process. I'm going to hopefully have time to start looking into adding it to my plugin (the one referenced) by the end of next week if you're interested

**Comment by oarrabi** [comment]
Any new info about this issue, I really miss using it with vscode :(

**Comment by mat-mcloughlin** [comment]
Due to some fine work by fr1zle the beginnings have been added to the vs code plugin. Please pull down version 0.6

**Comment by oarrabi** [comment]
Hello @mat-mcloughlin, after upgrading I am getting an error with VSCode. 
I am suspecting something regarding my PATH settings.

Anyways at the moment I had to revert back to 0.5 since in 0.6 coloring doesn't work anymore.

![screen shot 2016-06-13 at 13 43 35](https://cloud.githubusercontent.com/assets/1461052/16007923/0d3b4b5e-316d-11e6-9773-e2623faf6b0d.png)
